Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/PropertyEditorNodes.h

```diff
@@ -102,6 +102,16 @@ namespace AZ::DocumentPropertyEditor::Nodes
         FinishedEdit,
     };
 
+    //! Container: A node representing a container instance.
+    struct Container : NodeWithVisiblityControl
+    {
+        // These notify callback attributes may be invoked when a container is modified and handled
+        // alongside other messages in the adapter's message handler.
+        static constexpr auto AddNotify = CallbackAttributeDefinition<void()>("AddNotify");
+        static constexpr auto RemoveNotify = CallbackAttributeDefinition<void(size_t /*index*/)>("RemoveNotify");
```

Comment:
I don't believe you have to comment out the variable in the function declaration.